src/wblib/figures/hifs.py

```diff
@@ -17,10 +17,12 @@ def get_latest_forecast_issue_time(briefing_time: pd.Timestamp):
     return issued_time
 
 
-def get_dates_of_five_previous_initializations(
+def get_dates_of_N_previous_initializations(
     issue_time: pd.Timestamp,
+    N_previous_forecasts: int=5,
```

Comment:
To follow PEP8 would also suggest changing the name of `N_previous_forecasts` to  `number`. Also think might be more readable.